src/main/java/org/openstreetmap/atlas/geography/atlas/raw/sectioning/README.md

```diff
@@ -0,0 +1,89 @@
+
+## Detailed Steps
+
+The sectioning process is comprised of the following steps:
+
+1. Assemble a DynamicAtlas that will be used for sectioning. The reason for a DynamicAtlas is to be able to capture all OSM Ways that go outside the Shard bounds being processed and also capture any intersections of the each of these ways. This is the driving force behind the guarantee of consistent edge identifiers. If we can see a way in its entirety as well as all of its intersecting ways, we can section it deterministically and consistently, irrespective of which shard we are looking from and apply this consistent view to the current shard being processed. The logic for building the DynamicAtlas is to grab the full Atlas for the initial shard, then proceed to exapnd out to surrounding shards if there are any edges bleeding over the initial shard boundary. One caveat to call out here is that the Sharding being passed in, **must** be the same Sharding that was used to generate the input sliced raw Atlas. If it's not, there may be inconsistencies in the understanding of the surrounding shards that are availabe.
+
+2. Stream all the input Lines and identify the ones that becomes Edges or Areas. If a Line is set to become an Edge, we identify all location-based Nodes for that Edge. A Node will be created in one of the 4 cases:
+	a. A self-intersection - there is a repeated non-consecutive intersection in the Edge.
+	b. Configuration-based sectioning dependent on tagging (ex: section at barriers). 
+	c. Intersection with another edge of the same layer
+	d. Intersection with another edge of a different layer that either starts or ends at point of intersection
+	
+3. Stream all the input Points and distinguish between the ones that are shape points of underlying Atlas entities versus ones that become separate Atlas Points in the final Atlas. Most Points will become shape points and will be tracked by the underlying PolyLine or Polygon representation of the Area, Line or Edge that it's a part of.
+
+4. Stream all the Lines that have been identified as Edges from step 2 and split them into one or more Edge. This step will take care of both open and closed edges and handle forward and reverse Edge creation. It's important to note that the logic for sectioning closed Edges is slightly more complex since we try to avoid sectioning at the arbitrary start location where the OSM Way was drawn from. Instead, we try to section at any intersecting location with other edges or if those don't exist, we have a single Edge that starts and stops at the first Location of the underlying PolyLine.
```

Comment:
Nit: "open and closed edges", "intersecting location with other edges" -> "open and closed Edges", "intersecting location with other Edges"